docker/linksets/.env

```diff
@@ -0,0 +1,9 @@
+#Setting some variable so we can ensure compose works consistently
+#These values dont have to be the same as what is in the dockerfile
+DATABASE_URL=postgis
+S3_SOURCE_DATA_PATH=/source-data
```

Comment:
Move this to the `common.env` it will be useful for others I expect

Reply:
done